insights/parsers/tests/test_ironic_conf.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import print_function
+from insights.core.context import OSP
```

Comment:
You will see a flake8 error from this line if you run `flake8` from the root dir of your repo.